addons/docs/src/frameworks/angular/types.ts

```diff
@@ -24,6 +24,15 @@ export interface Component {
   rawdescription: string;
 }
 
+export interface Directive {
```

Comment:
In `@angular/core` a `Component` extends `Directive`  
https://github.com/angular/angular/blob/ea6a2e9f252f052664342d1613ae9fd59f1013c4/packages/core/src/metadata/directives.ts#L456

Since both interfaces are equal right now and a component in Angular is a Directive with more capabilities (+ a couple of extra fields in Component we don't use right now) we could just `export type Component = Directive`, right?

Reply:
Thanks for the feedback, I added the type alias. Let me know what you think! @kroeder @shilman